<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPasswordWindow</class>
 <widget class="QWidget" name="AddPasswordWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>142</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>375</width>
    <height>142</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(243, 242, 238);
border: 0px;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="title">
           <property name="font">
            <font>
             <family>Corbel</family>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>title:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="account">
           <property name="font">
            <font>
             <family>Corbel</family>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>account:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="password">
           <property name="font">
            <font>
             <family>Corbel</family>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>password:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="titleInput">
           <property name="font">
            <font>
             <family>Courier New</family>
             <pointsize>12</pointsize>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(197, 197, 197);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="accountInput">
           <property name="font">
            <font>
             <family>Courier New</family>
             <pointsize>12</pointsize>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(197, 197, 197);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="passwordInput">
           <property name="font">
            <font>
             <family>Courier New</family>
             <pointsize>12</pointsize>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(197, 197, 197);</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="workEndBtn">
         <property name="font">
          <font>
           <family>Corbel Light</family>
           <pointsize>11</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancel">
         <property name="font">
          <font>
           <family>Corbel Light</family>
           <pointsize>11</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
